src/main/java/interface_adaptors/update_price/UpdateController.java

```diff
@@ -14,13 +14,13 @@ public UpdateController(UpdatePriceInputBoundary useCase){
      * The controller packages the data and sends it off to the Update Use Case
      * @param serialNum the serial number of the item
      * @param price the price to update(for manager only)
-     * @return a String for the View  to present
      */
 
-    public String updateItem_price(String serialNum, double price, boolean updatePrice){
+    public void updateItem_price(String serialNum, double price, boolean updatePrice){
```

Comment:
Good changes here. I realized it doesn't make sense for the controller to return something back to the screen. Instead, it should be the presenter's job to interact indirectly with the screen.